tracer/build/_build/Build.Steps.cs

```diff
@@ -838,6 +865,51 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 workingDirectory / versionedName);
         });
 
+    Target ZipMonitoringHomeForAwsLambda => _ => _
+        .Unlisted()
+        .After(BuildTracerHomeForAwsLambda)
+        .Requires(() => IsLinux && (TargetPlatform == MSBuildTargetPlatform.x64 || TargetPlatform == ARM64TargetPlatform))
+        .Executes(() =>
+        {
+            var workingDirectory = ArtifactsDirectory / $"linux-{UnixArchitectureIdentifier}";
+            EnsureCleanDirectory(workingDirectory);
+
+            var arch = TargetPlatform == MSBuildTargetPlatform.x64 ? "amd64" : "arm64";
+            var archiveFile = workingDirectory / $"dd-trace-dotnet-aws-lambda-{arch}.zip";
+
+            // we want the tracer home directory to be in "/opt/datadog" in the AWS Lambda layer,
+            // so we need to nest it under a "datadog" folder in the zip file
+            var zipDirectory = MonitoringHomeDirectory / "lambda";
```

Comment:
I don't think we should be doing this work inside `MonitoringHomeDirectory`, as this will likely break other stages if run on the same filesysyem.

Reply:
I'm moving this into the artifacts directory instead, as you suggested elsewhere.